test/functional/feature_fee_estimation.py

```diff
@@ -128,16 +128,22 @@ def make_tx(wallet, utxo, feerate):
         fee_rate=Decimal(feerate * 1000) / COIN,
     )
 
+def check_fee_estimates_btw_modes(node, expected_conservative, expected_economical):
+    fee_est_conservative = node.estimatesmartfee(1, estimate_mode="conservative")['feerate']
+    fee_est_economical = node.estimatesmartfee(1)['feerate']
+    assert_equal(fee_est_conservative, expected_conservative)
+    assert_equal(fee_est_economical, expected_economical)
+
 
 class EstimateFeeTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 3
         # whitelist peers to speed up tx relay / mempool sync
         self.noban_tx_relay = True
         self.extra_args = [
-            [],
-            ["-blockmaxweight=68000"],
-            ["-blockmaxweight=32000"],
+            ["-datacarriersize=1000"],
+            ["-blockmaxweight=68000", "-datacarriersize=1000"],
+            ["-blockmaxweight=32000", "-datacarriersize=1000"],
```

Comment:
The test still works if no target weight is specified, and I can't think of why it would be necessary - is there a need for it? Otherwise, removing that + minimizing the extra config options would be preferable.

Reply:
The rationale was to fill up `node[2]` block template before mining, but you are right, this is not needed.
removed.